modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/context/index.jsx

```diff
@@ -98,6 +96,69 @@ const AppContextProvider = ({children}) => {
 			}
 		};
 
+		const getUserProjectAccess = async (userAccount, projectExternalReferenceCode) => {
+			let userProjectAccess = !!userAccount.organizationBriefs;
+			let denyAccess = false;
+
+			const organizationBriefs = userAccount.organizationBriefs;
+
+			if (organizationBriefs) {
+				try {
+					const {data: dataAccountOrg} = await client.query({
+						query: getAccountByExternalReferenceCodeOrganizations,
+						variables: {
+							externalReferenceCode: projectExternalReferenceCode
+						}
+					});
+
+					if (dataAccountOrg) {
+						const accountOrganizations = dataAccountOrg.accountByExternalReferenceCodeOrganizations?.items;
+
+						const organizationAccess = organizationBriefs.filter(
+							(organizationBrief) => (
+								accountOrganizations.some(
+									(accountOrganization) => accountOrganization.name === organizationBrief.name
+								)
+							)
+						).length > 0;
+
+						userProjectAccess = organizationAccess;
+					}
+				}
+				catch (error) {
+					const message = error.message;
+
+					if (!message.includes('(/accountByExternalReferenceCodeOrganizations) : null')) {
+						console.error(error);
+					}
+
+					denyAccess = true;
+				}
+			}
+
+			const accountAccess = !!userAccount.accountBriefs?.find(
+				(accountBrief) =>
+					accountBrief.externalReferenceCode ===
+					projectExternalReferenceCode
+			);
+
+			if (accountAccess) {
+				userProjectAccess = accountAccess;
+			}
+
+			const curUserProjectAccess = {
+				hasProjectAccess: userAccount.isOmniAdmin || userProjectAccess,
+				denyAccess
+			};
+
+			dispatch({
+				payload: curUserProjectAccess,
+				type: actionTypes.UPDATE_USER_PROJECT_ACCESS
+			});
+
+			return curUserProjectAccess;
```

Comment:
```suggestion
			const currentUserProjectAccess = {
				hasProjectAccess: userAccount.isOmniAdmin || userProjectAccess,
				denyAccess
			};

			dispatch({
				payload: currentUserProjectAccess,
				type: actionTypes.UPDATE_USER_PROJECT_ACCESS
			});

			return currentUserProjectAccess;
```

Reply:
In other parts of Liferay (including backend), we actually use cur[VariableName] instead of current, but I can change it to keep the consistency with CP stuff. I do think we eventually should use similar naming conventions as other parts of Liferay though.